test/pjrt/test_runtime.py

```diff
@@ -14,7 +15,17 @@
 class TestExperimentalPjrt(parameterized.TestCase):
 
   def setUp(self):
-    xr.set_device_type('CPU')
+    if xr.device_type() == 'CUDA':
+      xr.set_device_type('CUDA')
```

Comment:
Is this needed if `xr.device_type() == 'CUDA'`?

Reply:
good catch. It seems I need to add my test to test_runtime_gpu.py